styles/variables.scss

```
@@ -2,3 +2,22 @@
 $primary-color: #ff9900;
 $secondary-color: #20008d;
 $ternary-color: #dddddd;
+
+//Font Sizes Variables
+$font-title: 60px;
+$font-h1: 48.83px;
+$font-h2: 39.06px;
+$font-h3: 31.25px;
```

its interesting to avoid decimal numbers for css dimension and spacing, round to the closest integer
```suggestion
$font-h1: 49px;
$font-h2: 39px;
$font-h3: 31px;
```